dlgr/griduniverse/demo.py

```diff
@@ -11,6 +11,6 @@
     num_dynos_worker=participants,
 )
 
-results = experiment.analyze(data)
+results = experiment.average_score(data)
```

Comment:
Can you keep the `results = experiment.analyze(data)` API and instead have the `analyze` method return a JSON structure with results, e.g.:

```
{
    "average_payoff": 0.03,
    "average_score": 1.5,
}
```

The benefit of doing it this way is that it scales well with additional analyses and maintains a single API for all experiments that want to specify analyses. To replicate an experiment, you'd then run the following, regardless of the experiment:

```
import dallinger

experiment = dallinger.experiments.MyExperiment()
data = experiment.run()
results = experiment.analyze(data)
```